packages/vm/src/cache.rs

```diff
@@ -476,7 +479,7 @@ mod test {
     }
 
     #[test]
-    #[cfg(feature = "default-singlepass")]
+    #[cfg(feature = "metering")]
```

Comment:
Okay, so this code is just left as unmigrated placeholders and fails if metering flag is enabled now?
(Which is good to push into a separate PR)

Reply:
Jupp, at some point I realized that adding a simple metering flag makes almost everything compile and pass.